charts/celestia-local/files/scripts/start-bridge.sh

```diff
@@ -59,4 +59,6 @@ exec celestia bridge start \
   --gateway.port "$bridge_host_port" \
   --rpc.addr 0.0.0.0 \
   --rpc.port "$bridge_rpc_port" \
-  --keyring.accname "$validator_key_name"
+  --keyring.accname "$validator_key_name" \
+  --log.level "debug" \
+  --log.level.module "share/discovery:error"
```

Comment:
Cleaning up logs, and removing share discovery warnings about no networking, cleaner logs in case there is a celestia local issue.